src/topology/metric_space/basic.lean

```diff
@@ -803,6 +807,25 @@ begin
   apply ne_empty_of_mem B
 end⟩
 
+lemma mem_closure_range_iff {α : Type u} [metric_space α] {e : ℕ → α} {a : α} :
+  a ∈ closure (range e) ↔ ∀ε>0, ∃ k : ℕ, dist a (e k) < ε :=
+iff.intro
```

Comment:
In this lemma and the next, it doesn't matter that the domain of `e` is `nat`, does it?